doc/why_mf_next.md

```diff
@@ -0,0 +1,146 @@
+# Why `MessageFormat` needs a successor ([issue #49](https://github.com/unicode-org/message-format-wg/issues/49))
```

Comment:
In my opinion, the issue number should be placed along the article in a "more info" ,  "links" , "discussions" section or in the intro while giving context.